tests/interlocking_component/test_route_controller.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+from interlocking.interlockinginterface import Interlocking
+
+from src.interlocking_component.route_controller import RouteController
+
+
+class TestRouteController:
+    """This tests the route controller"""
+
+    @pytest.fixture
+    def route_controller(self) -> RouteController:
+        return RouteController()
+
+    def test_start_interlocking(self, route_controller):
+        route_controller.start_interlocking("test_example.ppxml")
```

Comment:
I think we should be consistent about referencing static files. The SimulationObjectUpdatingComponent wants the complete path as an argument and the RouteController only the filename of a specific folder. 
What do you prefer?

Reply:
The complete path is better, I will change it.